source/docs/api-reference/api-usage/faults.rst

```diff
@@ -5,8 +5,40 @@ Device Faults
 
 Faults are stored in two fashions. There are "live" faults, which are reported in real-time, and "sticky" faults, which assert persistently and stay asserted until they are manually cleared (like trouble codes in a vehicle).
 
-Sticky Faults can be cleared in Phoenix Tuner X, while a regular fault can only be cleared when the offending problem has been resolved. Faults can be cleared via the :guilabel:`Blink/Clear Faults` button.
+Sticky Faults can be cleared in Phoenix Tuner X or via the ``clearStickyFaults()`` function on the offending device, while a regular fault can only be cleared when the offending problem has been resolved. Faults can be cleared via the :guilabel:`Blink/Clear Faults` button.
```

Comment:
This paragraph is weirdly structured after adding the API mention. Maybe something similar to:

Sticky Faults can be cleared by clicking the \:guilabel\:\`Blink/Clear Faults\` button in Phoenix Tuner X, or by calling \`\`clearStickyFaults()\`\` on the device in the robot program. A regular fault can only be cleared when the offending problem has been resolved.